app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/item/RegularMessageItem.kt

```diff
@@ -82,26 +69,24 @@ import com.wire.android.ui.home.conversations.model.messagetypes.asset.Restricte
 import com.wire.android.ui.home.conversations.model.messagetypes.audio.AudioMessage
 import com.wire.android.ui.home.conversations.model.messagetypes.image.ImageMessageParams
 import com.wire.android.ui.home.conversations.model.messagetypes.location.LocationMessageContent
+import com.wire.android.ui.theme.Accent
 import com.wire.android.ui.theme.wireColorScheme
 import com.wire.android.ui.theme.wireTypography
 import com.wire.android.util.launchGeoIntent
 import com.wire.kalium.logic.data.asset.AssetTransferStatus
 import com.wire.kalium.logic.data.asset.isSaved
-import com.wire.android.ui.theme.Accent
 import com.wire.kalium.logic.data.id.ConversationId
 import com.wire.kalium.logic.data.user.UserId
 import kotlinx.collections.immutable.PersistentMap
 
 // TODO: a definite candidate for a refactor and cleanup
 @Suppress("ComplexMethod")
-@OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun MessageItem(
+fun RegularMessageItem(
```

Comment:
suggestion: this shouldn't be called "MessageItem" anymore as it doesn't contain everything that the message have - avatar is in a different composable now so if we want to split the `MessageItem` into two composables, both should have a meaningful names